### ISSUE TITLE
Saving files rapidly causes parse error

### ISSUE THREAD

**Issue by Oblongmana** [joeferraro/MavensMate-SublimeText#187]
The only possible slightly unusual thing I can think of that I did here was to hit save on one file rapidly after another, but I haven't seen this error in similar situations in this upgrade. The error also seems to include a local reference, which is interesting maybe?

Running mm server 0.35, MavensMate for Sublime Text v3.0.2

Just hit save again afterwards and it went perfectly fine.

==============================================
Compiling: DateBanked.page
Timestamp: Tue, 13 Aug 2013 15:55:24
Result: [OPERATION FAILED]: Whoops, unable to parse the response. Please report this issue at https://github.com/joeferraro/MavensMate-SublimeText
[RESPONSE FROM MAVENSMATE]: Traceback (most recent call last):

  File "<string>", line 301, in <module>

  File "<string>", line 53, in main

  File "<string>", line 104, in compile_selected_metadata

  File "/Users/josephferraro/Development/Python/mavensmate/tools/pyinstaller-dev/mm/build/pyi.darwin/mm/out00-PYZ.pyz/lib.mm_project", line 338, in compile_selected_metadata

  File "/Users/josephferraro/Development/Python/mavensmate/tools/pyinstaller-dev/mm/build/pyi.darwin/mm/out00-PYZ.pyz/lib.mm_client", line 474, in compile_with_tooling_api

TypeError: list indices must be integers, not str

[RESPONSE FROM MAVENSMATE]: Traceback (most recent call last):

  File "<string>", line 301, in <module>

  File "<string>", line 53, in main

  File "<string>", line 104, in compile_selected_metadata

  File "/Users/josephferraro/Development/Python/mavensmate/tools/pyinstaller-dev/mm/build/pyi.darwin/mm/out00-PYZ.pyz/lib.mm_project", line 338, in compile_selected_metadata

  File "/Users/josephferraro/Development/Python/mavensmate/tools/pyinstaller-dev/mm/build/pyi.darwin/mm/out00-PYZ.pyz/lib.mm_client", line 474, in compile_with_tooling_api

TypeError: list indices must be integers, not str

**Comment by ruwanthalk** [comment]
Try to save same class/component form the SF itself. Then I found out that I have missed a static resource

**Comment by garyspencer** [comment]
Had this same issue trying to save my visualforce page.  

I was able to refresh from server, but save yielded this error.  In hindsight - This occured after my sandbox was refreshed.  I think the issue was that one of my classes was deleted after the refresh.  

I turned off compile with tooling, and tried to save my visualforce page again -- This time the error was 'XyzController does not exist.' (Cool, clear errors are fun to deal with)  I opened my local copy of the controller that 'no longer existed' and Saved it successfully.  Saved the page successfully.  Turned compile with tooling back on, and then save the page again -- it worked like a charm.

My errors may be isolated incidents.  I believe the main issue here is the lack of clarity in the error message I received.  If I get a chance I will dig into the API and see if I can pinpoint a logic defect.
